### PR TITLE
Create sample to String.toLowerCase

### PR DESCRIPTION
**Pull request by bloderxd**
https://youtrack.jetbrains.com/issue/KT-20357

### CODE REVIEW

**Comment by bloderxd** [comment]
@ilya-g Could you review again, please?

File: libraries/stdlib/jvm/src/kotlin/text/StringsJVM.kt [hunk 1]
```diff
@@ -94,12 +94,16 @@ public actual fun String.replaceFirst(oldValue: String, newValue: String, ignore
 
 /**
  * Returns a copy of this string converted to upper case using the rules of the default locale.
+ *
+ * @sample samples.text.Strings.toUpperCase
```
Comment (ilya-g) [inline libraries/stdlib/jvm/src/kotlin/text/StringsJVM.kt]: There's `expect` counterpart and another `actual` implementation for JS, it's worth to place a sample reference there too.

**Comment by bloderxd** [comment]
@ilya-g What do you think now?

**Comment by ilya-g** [comment]
Squashed and merged, thank you for the PR

**Pull request merged**